apps/wallet-dashboard/app/(protected)/migrations/page.tsx

```diff
@@ -36,66 +40,63 @@ function MigrationDashboardPage(): JSX.Element {
     const queryClient = useQueryClient();
     const iotaClient = useIotaClient();
 
+    const [selectedStardustObjectsCategory, setSelectedStardustObjectsCategory] = useState<
+        StardustOutputMigrationStatus | undefined
+    >(undefined);
+
+    const { data: stardustMigrationObjects, isLoading } = useGetStardustMigratableObjects(address);
+    const { migratableBasicOutputs, migratableNftOutputs } = stardustMigrationObjects || {};
+
     const {
-        migratableBasicOutputs,
-        unmigratableBasicOutputs,
-        migratableNftOutputs,
-        unmigratableNftOutputs,
-    } = useGetStardustMigratableObjects(address);
+        totalIotaAmount,
+        totalNativeTokens: migratableNativeTokens,
+        totalVisualAssets: migratableVisualAssets,
+        totalUnmigratableObjects,
+    } = useMemo(() => {
```

Comment:
IMO its not worth to put a useMemo here, the cost of caching `stardustMigrationObjects` is worse than the benefit of memoizing this